jest.config.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
   transform: {
-    "^.+\\.svelte$": "svelte-jester",
+    "^.+\\.svelte$": ["svelte-jester", { preprocess: true }],
     "^.+\\.js$": "babel-jest",
   },
   moduleFileExtensions: ["js", "svelte", "json"],
-  testPathIgnorePatterns: ["/node_modules/", "/public/", "/storybook-static/"],
+  testPathIgnorePatterns: ["/node_modules/", "/public/", "/.storybook"],
```

Comment:
I think `storybook-static` is what's generated by `npm run build-storybook`. I think it should still be in the ignore list.